static/app/views/automations/edit.tsx

```diff
@@ -72,8 +71,6 @@ function AutomationBreadcrumbs({automationId}: {automationId: string}) {
 export default function AutomationEdit() {
   const params = useParams<{automationId: string}>();
 
-  useWorkflowEngineFeatureGate({redirect: true});
```

Comment:
We also call this on `DetectorViewContainer` which is the top level component for automations and detectors, so I think we need to remove it on that file as well